src/main/kotlin/com/pubnub/api/PubNubError.kt

```diff
@@ -131,6 +131,11 @@ enum class PubNubError(private val code: Int, val message: String) {
         "User name is missing"
     ),
 
+    TOKEN_MISSING(
+        152,
```

Comment:
I know that codes are not consistent across SDKs, but could you please check this number is free on Java SDK as well?

Reply:
I've used the first free number on Java which is `168`